pandas/_libs/tslib.pyx

```diff
@@ -216,8 +244,12 @@ def _test_parse_iso8601(ts: str):
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def format_array_from_datetime(ndarray[int64_t] values, object tz=None,
-                               object format=None, object na_rep=None):
+def format_array_from_datetime(
+    ndarray[int64_t] values,
```

Comment:
possibly can make `const int64_t[:]`?

Reply:
I think should hold off on these types of changes until we get warnings to error, unless there is a known upside

Reply:
its the usage encouraged by the cython folks, supposed to be slightly faster.  very marginal